PeerCo/small_scale_self_consumed_solar.md

```diff
@@ -0,0 +1,269 @@
+
+
+### Requirements on the system during the project lifetime
+
+During the whole duration of the project, the PV asset owner shall demonstrate proof of ownership of the asset and continued operation of the self-consumption. This documentation shall be made via all of the following:
+
+- sharing of consumption data throughout the period,
+
+- in the case of usage of the environmental attribute in reporting:
+    - contractual renunciation to the possibility of transfer of the environmental assets to a third party.
+
+- in the case of sales of the environmental attribute to a third party:
+    - contractual transfer of the environmental attribute ownership of the self-consumed generation,
+    - carbon accounting for the site (location and market-based) should reflect the sale of the carbon reduction corresponding to the transfer of the certificate.
+
+
+### Requirements on the operating context
+
+The project country shall meet the following criteria:
+1. delivery of micro-solar is not a requirement for building development or to meet an enforced carbon quota,
+2. emission intensity data is available at the hourly (or sub-hourly) level for the national or local grid.
+
+In cases where this methodology is used to generate carbon credits, extra requirements shall be met:
+
+3. PV arrays are not installed on more than 10% of all buildings in the market of operation
+
+
+## Project boundary
+
+GHG sources included or excluded from the project boundary.
+
+| Source                     | Gas     | Included | Justification           |
+| -------------------------- |---------|----------|------------------------ |
+| Grid electricity displaced | $CO_2$  | Yes      | Main emission source    |
+| Grid electricity displaced | $NO_x$, $SO_x$  | No       | Emissions data is not sufficiently granular and available the half hour or hour |
+
+
+## Measurement
+
+This section introduces the underlying data requirements.
+
+### Asset data
+
+PV production data shall be acquired via automatic digital communication to PeerCo's cloud once the project is established. However, in the initial phase of setting up a project (qualifying the site, its data and quantifying the benefits of the scheme), offline extractions of the data in an open format (which may be CSV, XLSX, etc.) may be accepted.
+
+All self-consumption data (and related generation data) shall be quality controlled prior its usage to generate verified certificates. This verification shall include:
+1. the maximum PV generation is in line with the PV asset's nominal rating,
+2. self-consumption is always lower than the total consumption,
+3. no significant PV generation is happening when there is no solar irradiance (e.g. at night).
+
+Where data is missing, filling of the gaps sall be allowed under the following conditions:
+1. data is not missing more than 5% of the time,
+2. gaps are filled with data whose daily profile and magnitude is consistent with the rest of the asset generation,
```

Comment:
It'd be great to include some guidance on some proposed methods around filling those gaps!

Reply:
There will be different methods depending on how the gaps appear. How detailed should this methodology get in defined the acceptable methods or is this up to the verification party?

Reply:
My thought at this stage further to Stephen's input was to propose a few options in appendix. I just haven't written them in there yet.
@ssuffian - if you guys have some specific ones already, I'm happy to reference them here too. Probably not down to code details, but a reasonably sharp method description allowing to reproduce and assess specifically enough

Reply:
We don't have anything super specific at WattCarbon yet, as our approach has been to only mint EACs for hours that we have actual data.

I imagine for solar it might be safe to do something as simple as a linear interpolation if an individual gap is no more than an hour long, and for longer gaps possibly using a previous day, same hour value, and as a last resort, some sort of solar+weather model to predict? I think it'd be important to make sure that the output data indicates whether a value was measured or computed, as that might change the end value of that EAC.

I was at the most recent meeting for the LF Energy OpenEEMeter and they had something cool but a bit more complex related to auto-correlation, it might be worth it to reach out to them: https://lfenergy.org/projects/openeemeter/ if we want to come up with a general approach.

Reply:
Thanks @ssuffian for the input.

At this stage, given that others do not consider gap filling, that other methodologies (even OpenEEmeter has carefully avoided formalising this in their latest version so far) do not do it either, and that we want to keep the discussion focus on essential points (governance etc - rather than such technical details which can be divisive), we decided to withdraw the acceptance of missing data with a threshold. 

This is reflected in [this commit](https://github.com/wattcarbon/WEATS/pull/2/commits/177799c4ed442d2f816205e7677072b357e18e98).

@ssuffian, we may however want to address this in a future version, once we have the big picture agreements in place and gather experience as an alliance. Happy to take the discussion again when we get there